packages/app-store/salesforce/lib/CalendarService.ts

```diff
@@ -79,7 +79,7 @@ export default class SalesforceCalendarService implements Calendar {
           .sobject("Contact")
           .create({
             FirstName,
-            LastName,
+            LastName: LastName || "",
```

Comment:
Requires a last name but need to fallback in case the attendee only supplies a first name